karspexet/templates/menu.html

```diff
@@ -9,7 +9,7 @@
         {% if forloop.counter == forloop.revcounter|add:"-1" %}
     <li>
       <a href="/">
-        <img class="luva" src="{% static 'svg/luva-inkscape.svg' %}">
+        <img class="luva" src="{% static 'svg/luva-inkscape.svg' %}" alt="">
```

Comment:
If we should fix HTML warning, we should do it properly, right?

Reply:
Menar du att jag inte gör det här eller?

Reply:
Borde inte `alt` innehålla en beskrivning av bilden? Åtminstone "Kårspexluvan", eller något smartare.

Reply:
Jag tycker inte det känns helt nödvnädigt:

```html
<!-- BAD: image URL will be read out -->
<img src="/wp-content/themes/punky/images/logo.png">

<!-- BAD: adding useless cruft alt -->
<img src="/wp-content/themes/punky/images/logo.png" alt="Logo">

<!-- GOOD: empty alt is silent -->
<img src="/wp-content/themes/punky/images/logo.png" alt="">
```

Saxat från https://davidwalsh.name/accessibility-tip-empty-alt-attributes

Reply:
Vid närmare eftertanke borde det nog vara "Huvudsidan" eftersom det är dit länken går. Med en tom `alt` försvinner den här länken helt.